nautobot_device_lifecycle_mgmt/tables.py

```diff
@@ -75,6 +76,62 @@ class Meta(BaseTable.Meta):
         )
 
 
+class SoftwareNoticeTable(BaseTable):
+    """Table for SoftwareNotice list view."""
+
+    pk = ToggleColumn()
+    id = tables.LinkColumn(
```

Comment:
`LinkColumn` is deprecated in django-tables2, use a regular `Column` with `linkify=True` instead.